cpp/src/parquet/arrow/schema_internal.cc

```diff
@@ -110,19 +115,20 @@ Result<std::shared_ptr<ArrowType>> MakeArrowTimestamp(const LogicalType& logical
   }
 }
 
-Result<std::shared_ptr<ArrowType>> FromByteArray(
-    const LogicalType& logical_type, const ArrowReaderProperties& reader_properties) {
+Result<std::shared_ptr<ArrowType>> FromByteArray(const LogicalType& logical_type,
```

Comment:
I'm just wondering why not simply passing `const ArrowReaderProperties& reader_properties` to all these functions? If we will add a third parameter which also comes from the properties, then we should directly use it.